packages/core-database-postgres/lib/models/pool.js

```diff
@@ -0,0 +1,26 @@
+const Model = require('./model')
```

Comment:
This should not be part of `core-database-postgres` because you are hardcoding transaction pool logic into a package that is supposed to be decoupled from it.

If you want to store data in the postgres database and add migrations you should do that in your transaction pool package.